google-login-plugin/src/com/google/gct/login/stats/UsageTrackerManager.java

```diff
@@ -0,0 +1,61 @@
+package com.google.gct.login.stats;
+
+import com.intellij.ide.util.PropertiesComponent;
+import com.intellij.openapi.diagnostic.Logger;
+import com.intellij.util.PlatformUtils;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * Stores the user's choice to opt in/out of sending usage metrics via the Google Usage Tracker.
+ */
+public class UsageTrackerManager {
```

Comment:
Discussing this with @elharo I don't think we should be following the static method pattern for manager style methods. The UsageTrackerManager should use instance methods and probably be a singleton. Possible intellij enforced as a service.http://www.jetbrains.org/intellij/sdk/docs/basics/plugin_structure/plugin_services.html  Although I'm not sure it applies.